project_euler/problem_77/sol1.py

```diff
@@ -0,0 +1,66 @@
+"""
+Prime summations
+Problem 77
+
+It is possible to write ten as the sum of primes in exactly five different ways:
+
+7 + 3
+5 + 5
+5 + 3 + 2
+3 + 3 + 2 + 2
+2 + 2 + 2 + 2 + 2
+
+What is the first value which can be written
+as the sum of primes in over five thousand different ways?
+"""
+
+
+def prime_sums(L=5000) -> int:
```

Comment:
- Please change the name of the function which returns the answer to `solution()` in line with #2786 
- Use descriptive variable names as mentioned in CONTRIBUTING.md
- Use type hints and doctests

Reply:
Thanks for your reply and help.

I've fixed it with 3e74825